Classes/Console/Install/Upgrade/UpgradeWizardExecutor.php

```diff
@@ -52,7 +59,7 @@ public function executeWizard(string $identifier, array $rawArguments = [], bool
         $wizardImplementsInterface = $upgradeWizard instanceof UpgradeWizardInterface && !$upgradeWizard instanceof AbstractUpdate;
         if ($force) {
             if ($wizardImplementsInterface) {
-                GeneralUtility::makeInstance(Registry::class)->set('installUpdate', $upgradeWizard->getIdentifier(), 0);
+                $this->registry->set('installUpdate', get_class($upgradeWizard), 0);
```

Comment:
Where did this change from getIdentifier to get_class come from? 🧐

Reply:
See in TYPO3: [UpgradeWizardsService](https://github.com/TYPO3/TYPO3.CMS/blob/f289c48e71b6def46af3435966344cd7b3a5f9df/typo3/sysext/install/Classes/Service/UpgradeWizardsService.php#L118) and [InstallerController](https://github.com/TYPO3/TYPO3.CMS/blob/0c531014d09d0b16a92438c86d717a33e0a345b8/typo3/sysext/install/Classes/Controller/InstallerController.php#L827)

Reply:
Yeah, good catch. I sneaked in this change, but is actually a bugfix, since TYPO3 never stores the identifier in the registry, but always the class name.

Reply:
In the long run our code here should use the TYPO3 API to mark wizards as done. I postponed that to another change though